lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
```

Comment:
You've repeated yourself a little for each time: a better approach would be to store all types in an object with the corresponding function. E.g:

```
const idTypes = {
  shabad: () => findId( Shabads, 3 )
  line: () => findId( Lines, 4 )
}

// your function

// Retrieve the correct generator function for our type
const generateId = idTypes[ type ]

// only need one loop now
console.log( generateId() )
```

This way, it's extremely simple and maintainable.